### ISSUE TITLE
Changing resolution in Beetle PSX HW causes a crash 

### ISSUE THREAD

**Issue by kurayami6** [libretro/beetle-psx-libretro#302]
Windows 7
i5
GTX 970
Beetle PSX HW 0.9.44.1 7E585CB

![capture](https://user-images.githubusercontent.com/31968468/36165028-5bcc12c4-1142-11e8-8227-f5df1c6783e4.PNG)

**Comment by Papermanzero** [comment]
Every change in opengl causes a crash

**Comment by simias** [comment]
The first step would be to bisect to figure out where the regression occurred.

**Comment by Papermanzero** [comment]
Then it would be helpful to provide older builds like dolphin, pcsx2, ppsspp etc are doing.

**Comment by simias** [comment]
I've just tried with the latest git version (1a9ad36e4fac85c0f4d986e2b4c02afda53c1b73) and `HAVE_OPENGL = 1` and I can't seem to get it to crash on linux with an Nvidia GTX 1080 using the proprietary driver.

I've tried both with software renderer and GL, switching back and forth between fullscreen and windowed using the "Use Fullscreen Mode" option in the "Video" menu. I've tried both with borderless fullscreen:

![fullscreen](https://user-images.githubusercontent.com/7925386/36331988-8228b1fe-1370-11e8-97a1-8b40ca351763.jpg)

**Comment by simias** [comment]
I do get spooky messages in the terminal though:

```
[WARN] Attrib array buffer was overflown!                                                            
[WARN] Attrib array buffer was overflown!                                                                                  
[WARN] Attrib array buffer was overflown!                                                                 
[WARN] Attrib array buffer was overflown!                                                  
[WARN] Attrib array buffer was overflown!                            
[WARN] Attrib array buffer was overflown!                                               
[WARN] Attrib array buffer was overflown!        
```

I think an attrib array buffer was overflown or something.

**Comment by Papermanzero** [comment]
Maybe it is related to Ram. i only have 2GB with my 760GTX. If the buffer exceeds the size, the emulator will crash.

**Comment by simias** [comment]
My big problem at the moment is that nothing makes sense. The only changes I've made of late relate to the core emulation code, and if you don't enable them the behaviour should be exactly the same. I can't comprehend how that could cause a regression in the GL code.

**Comment by twinaphex** [comment]
Well it definitely does happen on Windows with an nvidia GPU, that much is for certain.

**Comment by OptimalJew** [comment]
Latest commit fixed it, along with #300. Touching Internal GPU Resolution, Texture Filtering, or Display Full VRAM no longer crashes, and swapping between windowed and fullscreen with F works as well.
